exonum-java-binding/core/src/test/java/com/exonum/binding/core/runtime/JavaArtifactNamesTest.java

```diff
@@ -16,34 +16,22 @@
 
 package com.exonum.binding.core.runtime;
 
+import static com.exonum.binding.core.runtime.JavaArtifactNames.checkNoForbiddenChars;
 import static org.junit.jupiter.api.Assertions.assertThrows;
 
 import org.junit.jupiter.api.Test;
-import org.junit.jupiter.params.ParameterizedTest;
-import org.junit.jupiter.params.provider.ValueSource;
 
 class JavaArtifactNamesTest {
```

Comment:
These tests no longer seem to have anything to do with artifact names, hence I suggest to rename the class and methods.